src/routes/(console)/project-[region]-[project]/messaging/providers/settingsFormInput.svelte

```diff
@@ -124,6 +124,7 @@
         on:invalid={handleInvalid}
         extensions={['json']}
         bind:files={files[input.name]}
+        maxSize={10000000}
```

Comment:
Should this be hardcore?

Reply:
The text value for this was 10MB, I don't see why this shouldn't be 10MB for a JSON file. I also don't think the BE has any flag for this. Is there?

Reply:
Oh this is the firebase service json! I think the console actually reads the file and sends JSON.